spec/api-crash-reporter-spec.js

```diff
@@ -259,8 +259,11 @@ describe('crashReporter module', () => {
   describe('getLastCrashReport', () => {
     it('correctly returns the most recent report', () => {
       const reports = crashReporter.getUploadedReports()
-      const lastReport = reports[0]
+      const lastReport = crashReporter.getLastCrashReport()
+
+      // In our case, the first report is actually the newest
       assert(lastReport != null)
+      assert(lastReport === reports[0])
```

Comment:
It's not clear from this block that this is testing the new code.

Would it be possible to add an `assert(reports.length > 1)` to ensure comparison happened?